docs/content/setup/docker.md

```diff
@@ -25,6 +25,6 @@ sudo docker run --name monkey-island --network=host -d guardicore/monkey-island:
 
 There's no "upgrade-in-place" option for Docker. To get the new version, download it, stop the current container, and run the installation commands again with the new file.
 
-If you'd like to keep your existing configuration, you can export it to a file by using the Export button and then import it to the new server.
+If you'd like to keep your existing configuration, you can export it to a file by using the export button and then import it to the new server.
```

Comment:
Waiting on a resolution to https://github.com/guardicore/monkey/pull/934/files#r562772695